src/main.js

```diff
@@ -0,0 +1,65 @@
+/**
```

Comment:
Might be a bit confusing if main is not the page of the app but the sidebar?
Maybe call this sidebar and leave main free to the "main page" which is apps/activity ?